packages/admin-stories/src/docs/form/Layout.stories.mdx

```diff
@@ -0,0 +1,104 @@
+import { Meta, Canvas, Story } from "@storybook/addon-docs";
+
+<Meta title="Docs/Form/Layout" />
+
+# Form Layout
+
+Different types of layouts can be achieved, by adding `variant="horizontal"`, `variant="vertical"` or `fullWidth` to your field's props.
+
+When using `variant="vertical"` (which is the default value) the label is shown above the field. This is usually the most compact layout type.  
+With `variant="horizontal"` the label is shown on the left side of the field, where all labels have a fixed width.  
+Adding the `fullWidth` prop will cause the field and it's input to take up all the width, given by the parent element.
+
+<br />
```

Comment:
Do we need Breaks in Markdown 🤔

Reply:
We don't **need** them, but they make it look nicer 😄

If we strictly want to follow the markdown rules, we could probably add some custom css somewhere to make headlines have more margin or something...

Reply:
@johnnyomair I removed all `<br />` in markdown and styled the headlines instead.